plugin/github_link_opener.vim

```diff
@@ -12,7 +12,10 @@ function! s:OpenGitHubLink()
   let chars = '[-a-z0-9\.]\+'
   let path = matchstr(word, chars . '/' . chars)
   let has_more_than_one_slash = word =~# '/.\+/'
-  if empty(path) || has_more_than_one_slash
+  if &ft=='go' && has_more_than_one_slash
```

Comment:
It's a little silly on Vim's part, but Vim will take the user's settings into account when comparing. (You can read more about it [here](http://learnvimscriptthehardway.stevelosh.com/chapters/22.html).)

I think this will probably be fine no matter what the user's settings are, but just to be safe, can you change `==` to `==#` (case-sensitive)?

Reply:
Ah, I really should read that book. I pushed an update.